# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxWindowsPlatform.h
# Commit: 0facab7b9a1d
# Full Hash: 0facab7b9a1dad599c9f5049cdf104926eace696
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2022-12-21 21:21:23
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r 4f9ee3537468 -r 0facab7b9a1d gfx/thebes/gfxWindowsPlatform.h
--- a/gfx/thebes/gfxWindowsPlatform.h	Tue Dec 20 22:47:52 2022 +0000
+++ b/gfx/thebes/gfxWindowsPlatform.h	Tue Dec 20 22:47:53 2022 +0000
@@ -200,7 +200,13 @@
 
  protected:
   bool AccelerateLayersByDefault() override { return true; }
+
   nsTArray<uint8_t> GetPlatformCMSOutputProfileData() override;
+
+ public:
+  static nsTArray<uint8_t> GetPlatformCMSOutputProfileData_Impl();
+
+ protected:
   void GetPlatformDisplayInfo(mozilla::widget::InfoObject& aObj) override;
 
   void ImportGPUDeviceData(const mozilla::gfx::GPUDeviceData& aData) override;